### DISCUSSION TITLE
Artisan reading ET/BT temp incorrectly

### DISCUSSION THREAD

**Discussion by Freesobe** [artisan-roaster-scope/artisan#1203] [General]
Hello,
   I’m running a Dongyi BY-1 roaster on Artisan 2.8.2 from a new MacBook Pro.  I’ve configured to the correct preset but my ET/BT is reading wrong. 
   Example:  when the roaster is at 70F Artisan reads 44F. Every 10 degrees the roaster climbs, Artisan adds 1 or 2 degrees. 80F on roaster reads 45 on Artisan. 90F on roaster reads 46 on artisan. After the roaster hits 100 the jump is 2 degrees for every 10.
Has anyone had this problem and Does anyone know how to fix the issue?

thank you

**Comment by poundy** [comment]
my first reaction is this is the C vs F temperature, the roaster is reporting in C but you havae Artisan set to expect it in F. But the raw numbers don't match that. 
Send your setup file. Save out your current settings, rename the file with a .txt extension, and drag that here in the web (not via email)

**Reply by poundy** [reply]
Also I presume that you are only just setting this up, you've never used Artisan before or had instrumentaiton reading from this roaster?
